src/parser/postgresparser.cpp

```diff
@@ -219,7 +222,10 @@ parser::TableRef* PostgresParser::FromTransform(List* root) {
       result = RangeSubselectTransform(reinterpret_cast<RangeSubselect*>(node));
       break;
     }
-    default: { LOG_ERROR("From Type %d not supported yet...", node->type); }
+    default: {
+      LOG_ERROR("From Type %d not supported yet...", node->type);
+      throw NotImplementedException(" ");
```

Comment:
All other exceptions also have the error messages printed using LOG_ERROR above them.